update FAQ with no-kubernetes (#12872)
kubernetes/minikube: site/content/en/docs/faq/_index.md

minikube profiles are meant to be isolated from one another, with their own settings and drivers. If you want to create a single cluster with multiple nodes, try the [multi-node feature]({{< ref "/docs/tutorials/multi_node" >}}) instead.

> ## Can I use minikube as a Docker Desktop replacement ?

Reviewer: ```suggestion
## Can I use minikube as a Docker Desktop replacement?
```

> minikube's VM includes a Docker daemon running inside Linux for free, you could use 

Reviewer: ```suggestion
minikube's VM includes a Docker daemon running inside Linux for free, so you can use 
```

> `minikube docker-env` command to point your terminal's docker-cli to the Docker inside minikube.

Reviewer: ```suggestion
`minikube docker-env` to point your terminal's Docker CLI to the Docker inside minikube.
```

> You would need to start minikube with a VM driver (instead of docker) for example hyperkit on Mac and HyperV on windows

Reviewer: ```suggestion
You would need to start minikube with a VM driver, instead of docker, such as hyperkit on macOS and Hyper-V on Windows.
```

```
minikube start --driver=hyperv 
```

> Alternatively, you could use "minikube image build" command instead of "docker build"

Reviewer: ```suggestion
Alternatively, you can use `minikube image build`  instead of `minikube docker-env` and `docker build`.
```

> ## Can I start minikube without Kuberentes runninng ?

Reviewer: ```suggestion
## Can I start minikube without Kuberentes runninng?
```

> if you only want to use minikube as a Docker Desktop replacement and dont want to use kuberentes itself try:

Reviewer: ```suggestion
Yes! If you want to use minikube only as a Docker Desktop replacement without starting Kubernetes itself, try:
```

```
minikube start --no-kubernetes
```

> Alternatively if you temporary don't want to use Kubernetes you could use pause and later unpause Kubernetes 

Reviewer: ```suggestion
Alternatively, if you want to temporarily turn off Kubernetes, you can pause and later unpause Kubernetes 
```

```
minikube pause
```

> Alternatively you could use the auto-pause addon that will do that for you automatically.

Reviewer: ```suggestion
minikube also has an addon that automatically pauses Kubernetes after a certain amount of inactivity:
```
